app/ui-react/syndesis/src/modules/integrations/components/editor/endpoint/WithConfigurationForm.tsx

```diff
@@ -181,7 +190,12 @@ export const WithConfigurationForm: React.FunctionComponent<IWithConfigurationFo
           }
         >
           {() => (
-            <ConfigurationForm action={action} descriptor={data} {...props}>
+            <ConfigurationForm
+              action={action}
+              definitionOverride={webhookDefinitionOverride}
```

Comment:
```suggestion
              definitionOverride={isWebHook
    ? applyErrorKeysToForm(data, props.errorKeys!) : null}
```

I think would ensure the generated token `defaultValue` for "Context Path" will be supplied to the form.

Reply:
Check that `data` there is actually the definition, I might be wrong and this might be an object with the data shapes and `propertyDefinitionSteps` etc, in which case maybe an option is to pass a function down to `ConfigurationForm` that would allow you to do the `isWebHook` check and apply the definition override to the current page of the property definition, i.e. something like

```
definitionCustomizer={(definition) => isWebHook ? applyErrorKeysToForm(definition, props.errorKeys!) : definition}
```

where `definitionCustomizer` could be defined in `ConfigurationForm`'s property interface as `(definition: IConfigurationProperties) => IConfigurationProperties`

Reply:
You'll know you have the right behavior when this field has a hash in it:

![image](https://user-images.githubusercontent.com/351660/82369141-90d9af80-99e4-11ea-99c2-711befa641e4.png)

that's the symptom of ENTESB-13527